_specs_/websocket/paths/stocks/luld.yml

```
@@ -0,0 +1,26 @@
+summary: Limit-Up Limit-Down (LULD)
+description: |
+  Stream real-time LULD events for a given stock ticker symbol.
+responses:
+  200:
+    description: The WebSocket message for a LULD event.
+    content:
+      application/json:
+        schema:
+          $ref: '../../schemas/index.yml#/StockLULDEvent'
+        example:
+          ev: LULD
+          T: MSFT
+          h: 218.96
+          l: 198.11
+          i: ['*********** NEED VALUES ***********']
```

you can use `[21]` as an example value. About halfway down this page we list the different LULD indicators: https://polygon.io/glossary/us/stocks/conditions-indicators

Side note: not sure if we're planning on sprucing up that glossary page any time soon, but it would be cool to be able to link directly to certain sections (like the LULD indicators section) from our API documentation
```suggestion
          i: [21]
```